Data/dataset.txt

```diff
@@ -553,3 +553,92 @@ I'm a sock cutter and I cut socks. I'm a sock cutter and I cut socks. I'm a sock
 If Kantie can tie a tie and untie a tie, why can't I tie a tie and untie a tie like Kantie can.
 A black bloke's back brake-block broke.
 Big black bugs bleed blue black blood but baby black bugs bleed blue blood.
+Six Czech cricket critics.
```

Comment:
This one's nice (556) but its not a proper sentence [just a noun phrase] ... we can have some plain nounphrases too,  but just making a note here so that we don't let these overwhelm our dataset.. [since our task is to generate a tongue twister sentence]